### ISSUE TITLE
Replay Mode has no delay

### ISSUE THREAD

**Issue by ultioptimo** [WakingStones/CallersBane-Issues#5]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Server or Client?**
- e.g. Server

**Desktop (please complete the following information):**

**Additional context**

**Comment by jmb1304** [comment]
Can you please give some more information about this bug?

**Comment by ultioptimo** [comment]
yes. there is no delay when you spectate games you see them in real time which means cheating is very easy since you could in theory just create a second account watch your games and know your opponents cards.
